build/Settings.cs

```diff
@@ -211,6 +211,7 @@ public class SignInfo
                 "System.Diagnostics.PerformanceCounter.dll",
                 "System.IdentityModel.Tokens.Jwt.dll",
                 "System.Interactive.Async.dll",
+                "System.Memory.Data.dll",
```

Comment:
intentional?

Reply:
Yes; we treat DLLs outside of the Functions and WebJobs namespaces as third-party.